src/domain/entities/Note.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Note entity
+ */
+export default class Note {
```

Comment:
I think we can simply declare an interface here. Maybe we can replace them with classes later, but for now it will be more simplyl

Reply:
done